hibernate-core/src/main/java/org/hibernate/engine/spi/CollectionEntry.java

```diff
@@ -30,7 +30,10 @@
  * @author Gavin King
  */
 public final class CollectionEntry implements Serializable {
```

Comment:
Thanks, working on it. I copied the logic from `AssertionFailure`. I'll update that one too.